src/content/advanced/egress-ip-address-azure/index.md

```diff
@@ -46,15 +45,15 @@ spec:
   ...
 ```
 
-You can set the annotation to either an existing `Workload Cluster` or a brand-new one.
+You can set the annotation to either an existing workload cluster or a brand-new one.
+
+To enable and disable the use of an external IP address for an existing workload cluster, just add or remove the annotation from the `AzureCluster` resource at any time. Reconciliation takes just a few minutes to switch between the two modes and there shouldn't be any significant network packet loss while switching.
```

Comment:
Moved this from limitations, as it's not a limitation,  but in fact a nice feature.